### DISCUSSION TITLE
Disable text copying

### DISCUSSION THREAD

**Discussion by Electric-Dragon** [codex-team/editor.js#2062] [Q&A]
Hello I am going to be displaying some content that needs to be protected, so is there a way I can disable text copy and selection?

I have tried this but it does not stop text drag and drop or copying. Is there an EditorJS plugin for such a problem?

```
<div id="editorjs" style="-webkit-user-select: none; -ms-user-select: none; user-select: none;" onselectstart="return false" onpaste="return false;" onCopy="return false" onCut="return false" onDrag="return false" onDrop="return false"></div>

```

**Comment by MichalKeytrends** [comment]
Any update on this?
